src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int CIRCLE_DEFAULT_RADIUS = 10;
+    public static final String CIRCLE_DEFAULT_COLOR = Color.WHITE.name();
+    public static final int RANDOM_NUMBER = 4;
+    public static final int CONST_NUMBER = 50;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
```

Comment:
What is the difference between colorSupplier and random? Why the second one is 'final' and the first one no?